web/src/beta/components/fields/CameraField/index.tsx

```diff
@@ -0,0 +1,313 @@
+import { useCallback, useState } from "react";
+
+import Button from "@reearth/beta/components/Button";
+import NumberInput from "@reearth/beta/components/fields/common/NumberInput";
+import Icon from "@reearth/beta/components/Icon";
+import * as Popover from "@reearth/beta/components/Popover";
+import Slider from "@reearth/beta/components/Slider";
+import Text from "@reearth/beta/components/Text";
+import { useT } from "@reearth/services/i18n";
+import { styled, useTheme } from "@reearth/services/theme";
+
+import Property from "..";
+
+type CameraInput = {
+  name: string;
+  field: keyof CameraValue;
+};
+// Constants
+const CAMERA_XYZ: CameraInput[] = [
+  {
+    name: "Latitude",
+    field: "lat",
+  },
+  {
+    name: "Longitude",
+    field: "lng",
+  },
+  {
+    name: "Height",
+    field: "height",
+  },
+];
+
+const CAMERA_ANGLE: CameraInput[] = [
+  {
+    name: "Heading",
+    field: "heading",
+  },
+  {
+    name: "Pitch",
+    field: "pitch",
+  },
+  {
+    name: "Roll",
+    field: "roll",
+  },
+];
```

Comment:
Since we need to support internationalization with the names, these can't be constants. Can you move them in to the component and wrap in useMemo and wrap the names with i18n 🙏🏻

Reply:
oh yea, it skipped my mind. Thanks :+1:

Reply:
Done